lib/rspec/rails/matchers/active_job.rb

```diff
@@ -193,6 +193,9 @@ def queue_adapter
 
         # @private
         class HaveEnqueuedJob < Base
+          VERB_PRESENT_TENSE = 'enqueue'
```

Comment:
Note to self: `.freeze`

Reply:
I'm on the fence between using cryptic `self.class::VERB_PRESENT_TENSE` and defining methods in matchers, and calling them from the base class:
```
def verb_present_tense
  'enqueue'.freeze
end
```

Reply:
I mean naming wise they could be improved, but the reason is the later form causes new allocations even with the freeze I believe.

Reply:
Sorry, responded in the wrong thread.

I see three viable options:

1. keep as is (only add `.freeze`)

2. define methods with `freeze`
```ruby
def x
  'qwrqwfqw'.freeze
end
x.equal? x # => true
x.object_id == x.object_id # => true
```
Seems that some Ruby magic is going on, and it's being optimized to reduce allocations.
The downside is that we have two method definitions for no good reason, and that doubles the number of methods in those matchers.

3. define class-level `attr_reader`, and use `self.class.verb_present_tense`

Do you have a preference here?

Reply:
Speaking about the names, I'm out of good ideas.
`message_expectation_action`/`failure_message_expectation_action`?

Reply:
@pirj I am in favor of 1. Also more in favor of `message_expectation_action`. 😉

Reply:
Fixed!